wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+Other players' results can be checked by pressing their player tab.
+
+Personal match result placement will be privately sent under `#userlog` tab of the Chat Console.
+
+### Team Vs
+
+#### Interface
+
+![Team Vs Interface][Team Vs Interface image]
+
+_Team Vs interface_
+
+Team Vs mode pit two teams (Blue/Red) against each other based on Win Condition set.
+Fulfill the Win Condition criteria as closely as possible to reach the top spot in the team and compete with the other team by team total.
+
+Blue team members are placed at left side, red team members are placed at right side, and a crown at the center shows the current team supremacy.
+It will move left and right depending on which team fulfill the Win Condition most closely.
+
+When this mode was selected, all players in the Match Setup will receive either red or blue flag.
+The players can click on it to change from blue to red or vice versa.
+
+It is entirely possible to start a match with one colour team only, and there is no handicap bonus for the other team with lesser members.
+Placement of the players prior to the match does not matter; all blue members will be grouped to blue team and all red members will be grouped to red team.
+There is no role (like "Leader" or "Captain") distinction in the team; compete for both the team top spot and beating the other team score in total as a team.
+
+The surviving highest scoring member of either team will be shown as 1st at their respective team.
+Members in failed state will be located at the bottom of the team leaderboard and will not contribute to the team total as long as the members are in failed state.
+
+#### Result
+
+![Team Vs Grade screen][Team Vs Grade image]
+
+_Team Vs grade screen_
+
+In the results screen, declaration of match winner will be based on team total and announced at the center in winner's colour, with blue team total at the left and red team total at the right.
+
+**Only surviving members of the team's score will be counted**; it is possible for red team to be declared winner of the match when blue team has the superiority just because some blue team members providing the superiority were in failed state.
+
+The Team tab will show the total amount of hit score (300, 100, etc) done by the whole team and the average of the accuracy of the team.
+Combo not counted.
+Grade will not be shown on "Team" tab but will be shown at player's tab as watermark (if passed).
+
+### Tag Coop / Tag Team Vs
+
+#### Interface
+
+_Only available in osu! mode. **UNRANKED**_
+
+![Tag Coop Interface][Tag Coop Interface image]
+
+_Tag Coop interface_
+
+![Tag Team Vs Interface][Tag Team Vs Interface image]
+
+_Tag Team Vs interface_
+
+Tag Coop makes all players in the Match Setup work together to pass the beatmap, one combo at a time.
+Tag Team Vs is just Tag Coop with red/blue team competition.
+
+Do note that this team mode is only available in **osu! only** and **UNRANKED** (meaning, it will not be counted to ranked score).
```

Comment:
I would not use `**UNRANKED**` rather `_unranked_` (I mean, you aren't talkin' about the unranked image)